tests/benchmark/test_forecast_conformal_pilot.py

```
@@ -0,0 +1,288 @@
+    paths = write_forecast_conformal_pilot_report(
+        report,
+        json_path=out_json,
+        markdown_path=out_md,
+    )
+
+    assert paths == {"json": out_json, "markdown": out_md}
+    assert json.loads(out_json.read_text(encoding="utf-8"))["report_id"] == "writer"
+    assert "## Limitations" in out_md.read_text(encoding="utf-8")
+    with pytest.raises(ValueError, match="report must be a mapping"):
+        write_forecast_conformal_pilot_report("bad", json_path=tmp_path / "bad.json")  # type: ignore[arg-type]
+    with pytest.raises(ValueError, match="ForecastConformalPilot.v1"):
```

Ruff (0.15.15)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)